app/models/concerns/pdf_representable.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+module PdfRepresentable
+  extend ActiveSupport::Concern
+
+  def generate_pdf
+    raise "No authoritative_json to create PDF for #{oid}" unless authoritative_json
+    # temp_json_file = Tempfile.new("#{oid}_pdf_json")
```

Comment:
can we clean this up? or is there value here?

Reply:
Max and I cleaned this up.